### DISCUSSION TITLE
Curves, individual Points with Global Variables

### DISCUSSION THREAD

**Discussion by 3DRacer** [EdgeTX/edgetx#1191] [Ideas]
It would be easier if individual points of a curve could be adjusted using global variables. Maybe even scalable as an addition or multiplier to a set value. Then even several points in a curve could be changed proportionally.
Is there something already planned?

An example.
if you want to program a comfortable hover point adjustment for a copter with variable load, you have to do the following.

1. halve an existing expo curve. (we only want to adjust the center). so create an input from the gas stick with 50% travel and -50% offset.
2. create a mixer on the gas channel with that 50% input.
3. set the default expo curve in this mixer and add a GV as a setting.
4. now double the value in this mixer to 100% travel again. so set weight to 200% and 100% offset. 
this raises the question of whether the resolution has not been permanently halved.

To reconstruct something like this after some time is not easy.

How nice it would be if you could create the required curve yourself and make the desired settings via GV's directly in the curve.  A simple 3 point curve would be sufficient in this case which the middle point is a GV and not a fixed value.

**Comment by spiky1** [comment]
This would also be very convenient to adjust curves for gliders f.i. adjusting the elevator trim for butterfly in flight.
